packages/transforms/README.md

```diff
@@ -1 +1,55 @@
+# @endo/transforms
 
+> Source transforms for [SES](https://github.com/endojs/endo/tree/master/packages/ses)-enabled applications
+
+This package currently provides a transform to modify comments which would otherwise be rejected outright by SES.
+
+## Example
+
+```js
+// ESM example
+import { transformSource } from '@endo/transforms';
+import fs from 'node:fs/promises';
+
+const source = await fs.readFile('./dist/index.js', 'utf8');
+const sourceMap = await fs.readFile('./dist/index.js.map', 'utf8');
+const sourceUrl = 'index.js'; // assuming the source map references index.js
+const sourceType = 'script';
+
+const { code, map } = await transformSource(source, {
+  sourceMap,
+  sourceUrl,
+  sourceType,
+});
+
+await fs.writeFile('./dist/index.ses.js', code);
+await fs.writeFile('./dist/index.ses.js.map', JSON.stringify(map));
+```
+
+The censorship-evasion function can also be used directly, if you have parsed
+your source into an AST via Babel:
+
+```ts
+// TypeScript example
+import { evadeCensor } from '@endo/transforms';
+import { type Comment } from '@babel/types';
```

Comment:
Supposing we switched the internal implementation from Babel to Esprima, removing the Babel API surface would be a breaking change. I’m thinking it might be possible to anticipate that kind of migration, allowing users to continue upgrading minor versions during the transition, shifting from the Babel version to the Esprima on their own time, but only entraining the dependency on one or the other in any particular release. Perhaps “babel” could appear in the module specifier when the user explicitly elects that level of coupling.

Reply:
I think I'd prefer:

1. _not_ exporting the underlying `evadeCensor` function, thus
2. decoupling the public API from these implementations

As it currently stands, LavaMoat shouldn't need anything except the "main" function--which is the same as current usage in `@endo/bundle-source`.

If, in the future, someone needs a lower-level, implementation-specific API, we could provide it by splitting this package in two: expose these APIs in the new package then consume it from _this_ higher-level package.